src/frontend.Dockerfile

```
@@ -0,0 +1,26 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.5-214
+USER 0
+RUN dnf install -y openssh-clients git docker jq curl tar 
+RUN useradd -ms /bin/bash builder
+RUN mkdir -p /container_workspace
+RUN chown -R builder:builder /container_workspace
```

I would merge these `RUN` commands together so we don't add new layer each time we create user, folder and change owner of that folder.
```suggestion
RUN useradd -ms /bin/bash builder && \
    mkdir -p /container_workspace && \
    chown -R builder:builder /container_workspace
```